install/installer/pkg/components/node-labeler/deployment.go

```diff
@@ -71,6 +71,7 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 			},
 			*common.KubeRBACProxyContainerWithConfig(ctx),
 		},
+		Tolerations: common.WithTolerationWorkspaceComponentNotReady(ctx),
```

Comment:
What makes this required for node-labeler? The new taints should only apply to workspace and headless nodes.